aspnetcore/web-api/http-repl.md

```diff
@@ -809,7 +809,23 @@ To set an HTTP request header, use one of the following approaches:
 
 ## Test secured endpoints
 
-The HTTP REPL supports the testing of secured endpoints through the use of HTTP request headers. Examples of supported authentication and authorization schemes include basic authentication, JWT bearer tokens, and digest authentication. For example, you can send a bearer token to an endpoint with the following command:
+The HTTP REPL supports the testing of secured endpoints in two ways: via the default credentials of the logged in user or through the use of HTTP request headers. 
+
+### Default credentials
+
+Consider a scenario in which the web API you're testing is hosted in IIS and is secured with Windows authentication. You want the IIS application pool identity's credentials to flow across to the HTTP REPL requests. To pass the default credentials of the logged in user:
```

Comment:
I don't think this is exactly right. Setting the property would pass the credentials of the the user running the tool (on whatever machine they're on) to IIS (on the server they're sending the requests to). The application pool's identity isn't involved.  I think the second sentence should be
> You want the credentials of the user running the tool to flow across to the http endpoints being tested.

Or something along those lines.

Reply:
Agreed. But I do like this wording change in general.

Reply:
Agreed. But I do like this wording change in general.

Reply:
Agreed. But I do like this wording change in general.